vignettes/articles/modify-functions.Rmd

```diff
@@ -0,0 +1,197 @@
+---
+title: "Modifier Functions"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(gtsummary)
+theme_gtsummary_compact()
+```
+
+## Introduction
+
+Oftentimes when a table is created using {gtsummary} the default formatting does not exactly match a user's preferred formatting. To make modifications to tables after the table has been built, the `modify_*()` functions were created.
```

Comment:
Can we put each sentence on a single row? It makes maintaining the article easier in the future when there are changes

Reply:
Done